api/modules/currencyconverter/views.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from api.modules.currencyconverter.constants import CURRENCY_CONVERTER_API_URL
+from api.modules.currencyconverter.currencycon_item import CurrencyItem
+
+@api_view(['GET'])
+def get_currency_exchange_rate(request, source, target):
+
+
+    try:
+        query = source + "_" + target
+        api_response = requests.get(CURRENCY_CONVERTER_API_URL.format(query))
+        api_response_json = api_response.json()
+        if not api_response.ok:
+            error_message = "Missing parameters in request"
+            return Response(error_message, status=status.HTTP_400_BAD_REQUEST)
+
+        response = CurrencyItem(source=source,
```

Comment:
if `api_response.ok` is `False` then it is possible `api_response_json` won't have `query` index 
Add a check for that index